test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -52,7 +52,10 @@ export default class BasePageObject {
       }, '');
     }
     await this.t.navigateTo(`http://localhost:3000${this.url}${qs}`);
-    await this.t.expect(Selector(FORM_SELECTOR).exists).eql(true);
+
+    if (queryParams?.render !== false) {
+      await this.t.expect(Selector(FORM_SELECTOR).exists).eql(true);
+    }
```

Comment:
@leemchale-okta sometimes we actually don't want to render the widget for certain tests, so I'm adding a guard here to skip this assertion when that is the case

Reply:
Checking explicitly for `!== false` since sometimes it's `undefined`